### PR TITLE
chore (1-3076): add remaining "project resources" to status payload

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR adds member, api token, and segment counts to the project status payload. It updates the schemas and adds the necessary stores to get this information. It also adds a new query to the segments store for getting project segments.

I'll add tests in a follow-up.

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx [hunk 1]
```diff
@@ -97,25 +89,32 @@ const ListItem: FC<
     </ListItemRow>
 );
 
+const useProjectResources = (projectId: string) => {
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx:92]: Is that a common practice we do, having a subhook? I would have used project status hook directly.

File: src/lib/features/project-status/project-status-service.ts [hunk 2]
```diff
@@ -1,28 +1,66 @@
...
+            this.apiTokenStore
+                .getAll()
+                .then(
+                    (tokens) =>
+                        tokens.filter(
+                            (token) =>
+                                token.project === projectId ||
+                                token.projects.includes(projectId),
+                        ).length,
+                ),
```
[the remark this answers is not in the document]
Comment (sjaanus) [reply src/lib/features/project-status/project-status-service.ts:50]: Yes, makes sense to have new method.

**Review by sjaanus — APPROVED** [review]

File: src/lib/features/project-status/project-status-service.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (thomasheartman) [reply src/lib/features/project-status/project-status-service.ts:50]: Cool! I'll address it in the followup.

File: frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx (hunk 1 above)
Comment (thomasheartman) [reply frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx:92]: I ... don't know? I think I've seen Mateusz do it a couple times, so .. yeah? In this case, I just thought it made the main component more legible. But I'm not married to it 💁🏼

**Pull request merged**